cypress/support/fragments/finance/budgets/budgets.js

```diff
@@ -99,7 +99,7 @@ export default {
       if (expenseClasses.length) {
         this.updateBudgetViaApi({
           ...resp,
-          _version: resp._version + 1,
+          // _version: resp._version + 1,
```

Comment:
Do you need this comment?

Reply:
this was in the previous version, but it prevented two tests from running, I will remove it in the next pr